src/monument/Monument.ts

```diff
@@ -0,0 +1,20 @@
+import { type MonumentLocation, type MonumentStructure } from "./types";
+
+class Monument implements MonumentStructure {
```

Comment:
¿Por qué hacéis copy/paste?

Reply:
> Es la misma clase que la que tenemos en el back y la necesitamos en el front

Eso no es verdad.

Reply:
Ok, lo intentamos implementar de otra manera